README.md

```diff
@@ -3,15 +3,25 @@
 Welcome to the Personal Travel Assistant project! This application is designed to assist you with your travel needs and make your journey more enjoyable.
 
 ## How to Run
+<ol>
+    <li>
+        <p>
+            Before you begin, make sure you have <a href="https://nodejs.org/en/download/current">Node.js</a> installed on your system.
+        </p>
+    </li>
+    <li>
+        <p>After that, go into the travel-assistant folder</p>
+        <code>cd .\travel-assistant\</code>
+    </li>
+    <li>
+        <p>and then you can set up the project with the following command:</p>
```

Comment:
Include the directory change in this line

Reply:
no

Reply:
yes

Reply:
these are 2 different steps. you either modify the repo as with the issue mentioned above, or let the readme like this.